web/docs/deployment/installation.md

```diff
@@ -56,6 +56,11 @@ In `vars.yml`, generate different random strings for these values:
 - `rabbitmq_password`
    * The password for the server and grader apps to connect to RabbitMQ.
 
+If you generate a different ssh key filename, edit the following variable in `vars.yml` to allow access to root:
+
+- `ansible_ssh_private_key_file`
```

Comment:
I think it makes more sense if this is located in the "F. Running Ansible playbooks" section instead, what do you think?

Reply:
in terms of practicality, I think your suggestion is better
developer will encounter the error in step `F. Running Ansible playbooks`, and they can find the solution immediately
let me edit it 🙇